### PR TITLE
Add get coordinates by place id

### PR DESCRIPTION
**Pull request by giolaza**

### CODE REVIEW

**Comment by freekmurze** [comment]
Can you add a test and document this function in the readme?

**Comment by giolaza** [comment]
@freekmurze done

File: README.md [hunk 1]
```diff
@@ -317,6 +317,140 @@ Geocoder::getCoordinatesForAddress('Infinite Loop 1, Cupertino');
...
+
+/*
+  This function returns an array with keys
+  Array
+  (
+    "lat" => 52.5102586
+    "lng" => 13.4218842
+    "accuracy" => ROOFTOP
+    "formatted_address" => Michaelkirchstraße 13, 10179 Berlin, Deutschland
+    "viewport" => stdClass Object
```
Comment (freekmurze) [inline README.md:335]: Could you clean this up, and remove the `stdClass` references (like in the others sections of the readme)?

**Review by freekmurze — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (giolaza) [reply README.md]: okey, used another sample

**Pull request closed without merging**